homeassistant/components/mill/climate.py

```diff
@@ -207,3 +212,114 @@ def hvac_modes(self):
     async def async_update(self):
         """Retrieve latest state."""
         self._heater = await self._conn.update_device(self._heater.device_id)
+
+class MillRoom(ClimateDevice):
+    """Representation of a Mill Room device."""
+    """Controls temperature on multiple heaters."""
+
+    def __init__(self, room, mill_data_connection):
+        """Initialize the thermostat."""
+        self._room = room
+        self._conn = mill_data_connection
+
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        return SUPPORT_FLAGS
+
+    @property
+    def available(self):
+        """Return True if entity is available."""
+        """Rooms doesn't have its own connection and are allways available."""
+        return True
+
+    @property
+    def unique_id(self):
+        """Return a unique ID."""
+        return self._room.room_id
+
+    @property
+    def name(self):
+        """Return the name of the room."""
+        return self._room.name
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        res = {}
+        return res
+
+    @property
+    def temperature_unit(self):
+        """Return the unit of measurement which this thermostat uses."""
+        return TEMP_CELSIUS
+
+    @property
+    def target_temperature(self):
+        """Return the temperature we try to reach."""
+        return self._room.comfort_temp
+
+    @property
+    def target_temperature_step(self):
+        """Return the supported step of target temperature."""
+        return 1
+
+    @property
+    def current_temperature(self):
+        """Return the current temperature."""
+        return self._room.avg_temp
+
+    @property
+    def fan_mode(self):
```

Comment:
Can you set fan mode on room level?

Reply:
I haven't really thought about it...
Out of my head I'd say yes and that we need to add code to set it corectly for all the relevant heaters in the room. 
I dont know what would happend if you try to set fan mode on a heater without fan.